uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/FloorFilterController.qml

```diff
@@ -38,18 +122,38 @@ QtObject {
 
     property bool autoselectSingleFacilitySite
 
+    property var automaticSelectionMode
+
     property FloorManager floorManager
 
     //refresh()?
     property string selectedLevelId
 
     property string selectedFacilityId
```

Comment:
🔧  Consider making these 

```qml
property readonly alias selectedLevelId: internal.selectedLevelId
``` 

This way they can only be set using your custom functions in the public interface.